src/contracts/atlas/Storage.sol

```diff
@@ -57,6 +58,7 @@ contract Storage {
 
         // Gas Accounting
         surcharge = msg.value;
+        surchargeRecipient = msg.sender;
```

Comment:
Let's rather set it using an address passed in as an arg on deployment - we might not want it to be the same as the deployer, and then we can avoid the transfer process